runtime/src/bank.rs

```diff
@@ -1305,8 +1320,9 @@ impl Bank {
         // if I'm the first Bank in an epoch, count, claim, disburse rewards from Inflation
 
         // calculated as: prev_slot / (slots / year)
-        let slot_in_year =
-            (self.epoch_schedule.get_last_slot_in_epoch(prev_epoch)) as f64 / self.slots_per_year;
+        let num_slots = self.epoch_schedule.get_last_slot_in_epoch(prev_epoch)
```

Comment:
hmm, I think this could overflow at the initial inflation?

I think for the activated epoch, this is true: `self.get_inflation_start_slot()` (= start slot of new epoch) > `self.epoch_schedule.get_last_slot_in_epoch(prev_epoch)` (= end slot of old epoch)

Reply:
Ah, thank you, I missed that case.
I think `saturating_sub` should be correct, right?